ansible/provision-and-execute-tests/playbook.yml

```diff
@@ -102,15 +106,36 @@
 - name: Provision Windows test executor instances and execute tests
   hosts: windows
   vars:
-    node_version: 16.3.0
+    node_version: 16.14.0
 
     # The following information is populated using the gathered inventory variables
     fluent_bit_package_name: "{{ tags.fb_package_name }}"
     pr_number: "{{ tags.pr_number }}"
+    fb_version: "{{ tags.fb_version }}"
+    os_distro: "{{ tags.os_distro }}"
+    os_version: "{{ tags.os_version }}"
+    arch: "{{ tags.arch }}"
+
+    new_relic_api_key: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_API_KEY') }}"
+    new_relic_account_id: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_ACCOUNT_ID') }}"
+    new_relic_region: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_REGION') }}"
```

Comment:
We should probably add a comment here explaining that these are injected in the environment during the Fargate task definition creation.